src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const errorMessage = 'Error';
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case `removeProperties`:
+        for (const value of action.keysToRemove) {
```

Comment:
would be better to call it so as we iterate over keys
```suggestion
        for (const key of action.keysToRemove) {
```

Reply:
fixed